src/components/DocumentationTopic/Hero/Hierarchy.vue

```diff
@@ -137,6 +142,11 @@ export default {
         console.error(`Reference for "${id}" is missing`);
         return all;
       }, []);
+      const immediateParent = last(parentTopics);
```

Comment:
The short answer is: I tried, and it broke things! 😅 The selected page in the navigator was wrong if i omitted the "overload group" page from a symbol's breadcrumb path. Instead of selecting the overload group page (which had the same name and is meant to be used as a stand-in for all the overloaded symbols), it selected the parent page of the overload group, creating an even more confusing situation. By leaving the overload group in the breadcrumb path, the correct page was highlighted in the navigator but it created this duplicate-name situation, hence this PR.